test/ceaser_cipher.test.js

```diff
@@ -0,0 +1,13 @@
+import cipher from '../src/ceaser_cipher';
+
+it('it should return a ciphered word', () => {
+  expect(cipher('abc', 1)).toBe('bcd');
```

Comment:
- [ ] Your test should have a case that wraps z to a. That way, you won't be missing out on anything.